src/main/kotlin/org/rust/lang/core/resolve/ref/RsMethodCallReferenceImpl.kt

```diff
@@ -105,9 +105,10 @@ data class MethodResolveVariant(
     override val element: RsFunction,
     /**
      * If the method defined in impl, this is the impl. If the method inherited from
-     * trait definition, this is the impl of the actual trait for the receiver type
+     * trait definition, this is the impl of the actual trait for the receiver type.
+     * Otherwise it's just a trait the method defined in
```

Comment:
I'd replace "this is the impl/trait" by "this contains impl/trait" because now `source` is container for `RsTraitOrImpl`